### PR TITLE
Remove minor bugs from docs.

### PR DESCRIPTION
**Pull request by Heaven31415**
Hey, I was reading your docs some time ago, really liked them. I have found some minor bugs and thought it would be a good idea to fix them.

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

File: templating.rst [hunk 1]
```diff
@@ -51,7 +51,7 @@ template - a text file parsed by PHP that contains a mix of text and PHP code:
 
 .. index:: Twig; Introduction
 
-But Symfony packages an even more powerful templating language called `Twig`_.
+But Symfony packages have even more powerful templating language called `Twig`_.
```
Comment (javiereguiluz) [inline templating.rst]: I think this change is not OK. "packages" here doesn't mean *"software libraries"* but the verb *"includes"*

**Review by javiereguiluz — APPROVED** [review]
@Heaven31415 thanks for fixing all this! I left a minor comment about one change.

File: templating.rst (hunk 1 above)
Comment (Heaven31415) [reply templating.rst]: Thanks, I fixed that.

**Comment by javiereguiluz** [comment]
@Heaven31415 thanks for fixing these typos. It was a great first contribution to Symfony Docs! Congrats on that! :tada: (Note: we merged in 4.1 branch and we'll merge it in the other branches later automatically)

**Pull request closed without merging**